bindings/python/iota_sdk/types/feature.py

```diff
@@ -81,6 +81,20 @@ class MetadataFeature:
     entries: Dict[str, HexStr]
 
 
+@json
+@dataclass
+class StateMetadataFeature:
+    """Defines metadata, arbitrary binary data, that will be stored in the output.
```

Comment:
I know we already had that in Rust but is there no better doc than that somewhere? Maybe in the TIP?

Reply:
That's not exceptional but at least it's a bit more accurate yeah